packages/application-generic/src/health/trigger-queue.health-indicator.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  HealthCheckError,
+  HealthIndicator,
+  HealthIndicatorResult,
+} from '@nestjs/terminus';
+import { Injectable } from '@nestjs/common';
+
+import { TriggerQueueService } from '../services';
+
+@Injectable()
+export class TriggerQueueServiceHealthIndicator extends HealthIndicator {
+  private INDICATOR_KEY = 'triggerQueue';
+
+  constructor(private triggerQueueService: TriggerQueueService) {
+    super();
+  }
+
+  async isHealthy(): Promise<HealthIndicatorResult> {
+    const runningStatus =
+      await this.triggerQueueService.bullMqService.getRunningStatus();
+
+    if (!runningStatus.queueIsPaused) {
```

Comment:
We only create a queue for the TriggerQueueService.

Reply:
I wasn't able to find in the Bull MQ API any other way to know if the queue was running besides this property and to check if the queue was instantiated.

Reply:
we should also check here is the worker is running by using the flag that you do return

Reply:
Unfortunately we can't do it as only a queue is created in `TriggerQueueService` therefore the worker comes as undefined. 🙁